src/gui/update.rs

```diff
@@ -230,6 +230,13 @@ pub fn handle_message(ajour: &mut Ajour, message: Message) -> Result<Command<Mes
 
             return Ok(Command::perform(select_directory(), message));
         }
+        Message::Interaction(Interaction::ResetColumns) => {
+            log::debug!("Interaction::ResetColumns");
+
+            ajour.config.column_config = Default::default();
+            apply_config(ajour, ajour.config.clone());
+            let _ = &ajour.config.save();
+        }
         Message::Interaction(Interaction::OpenLink(link)) => {
```

Comment:
@tarkah, currently when pressing the button and executing this code and closing and reopening Ajour the columns is reset, however I am wondering why it does not happen instantly and appear changed in GUI?

Reply:
Let me test and see if I can figure it out